target/linux/octeon/config-5.4

```diff
@@ -25,13 +25,8 @@ CONFIG_BLK_MQ_PCI=y
 CONFIG_BLK_SCSI_REQUEST=y
 CONFIG_BUILTIN_DTB=y
 # CONFIG_CAVIUM_CN63XXP1 is not set
-CONFIG_CAVIUM_OCTEON_CVMSEG_SIZE=2
-CONFIG_CAVIUM_OCTEON_LOCK_L2=y
-CONFIG_CAVIUM_OCTEON_LOCK_L2_EXCEPTION=y
-CONFIG_CAVIUM_OCTEON_LOCK_L2_INTERRUPT=y
-CONFIG_CAVIUM_OCTEON_LOCK_L2_LOW_LEVEL_INTERRUPT=y
-CONFIG_CAVIUM_OCTEON_LOCK_L2_MEMCPY=y
-CONFIG_CAVIUM_OCTEON_LOCK_L2_TLB=y
+CONFIG_CAVIUM_OCTEON_CVMSEG_SIZE=0
+# CONFIG_CAVIUM_OCTEON_LOCK_L2 is not set
```

Comment:
This changes quite a lot of options. I'd like to have that in a separate commit.

Probably this should be tested with other devices as well.

Reply:
sure, i dropped all the changes regarding disabling l2 cache locking.